src/instagram.js

```diff
@@ -1,110 +1,101 @@
+"use strict";
+
+exports.__esModule = true;
```

Comment:
not sure i understand this either (reference my other comment.). is it because of something automatic reformatting?

Reply:
Hey, running prettier via the vscode plugin added some unexpected markup. I reformatted and then manually aligned what was a miss, all eslint errors have been corrected, corrected merge conflicts with the new hashtag work.